src/main/java/seedu/duke/Parser.java

```diff
@@ -110,24 +111,27 @@ public static Command checkAction(String inputCommand) throws NumberFormatExcept
             if (isProjectListView) {
                 System.out.println("Not in Task View!"); //----------REPLACE WITH EXCEPTION
             } else {
-                for (int i = 1; i < inputs.length - 1; i++) { //Task name after task keyword and before date
+                for (int i = 1; i < inputs.length; i++) { //Task name after task keyword and before date
                     description += inputs[i];
                 }
+                commandType = new TaskCommand(description, projectIndex);
+            }
+            break;
+        case "deadline":
+            if (!isProjectListView) {
```

Comment:
require an else statement? We can implement the "deadline" function for projects as well. 
I would be able to do that since I need to combine it with printing out the statement during the "select" function.
If approved, I will create an issue for it and start working on it.

Reply:
The user guide has no deadline for projects. But if simple, can just do